Libraries/CO.CDP.Configuration/Helpers/ConnectionStringHelper.cs

```diff
@@ -27,7 +27,9 @@ public static string GetConnectionString(IConfiguration configuration, string na
         var database = connectionSettings["Database"];
         var username = connectionSettings["Username"];
         var password = connectionSettings["Password"];
+        var maxPoolSize = connectionSettings["MaxPoolSize"];
+        var maxPoolSizeString = string.IsNullOrWhiteSpace(maxPoolSize) ? "" : $"MaxPoolSize={maxPoolSize};";
```

Comment:
Could you please define a fallback limit here? It's better to ensure we operate within a defined range, rather than relying on other components to handle the absence of this parameter. This way, we can avoid unexpected behaviour if the environment variable isn't set.

Reply:
Yes they are defined as default in appsetting.json